app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextMonthYearField.kt

```diff
@@ -191,16 +192,38 @@ class EditTextMonthYearField(
     override fun isValid(): ValidationResult {
         val result = super.isValid()
         if (result.isValid) {
-            return validateUpperLimit() ?: result
+            when {
+                upperLimit != null -> return validateUpperLimit() ?: result
+                lowerLimit != null -> return validateLowerLimit() ?: result
+            }
```

Comment:
I think this should throws an exception when both limits are set, so that the developer will be aware about this issue. Or implement the validation for both limits.